experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -323,6 +325,11 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
     request.prependListener(
       'response',
       (response: http.IncomingMessage & { aborted?: boolean }) => {
+        // if there are no other response listeners, we don't wanna set this as there won't ever be any response
```

Comment:
IIUC, this can be better clarified as:

```suggestion
        // if there are no other response listeners, we don't wanna set this as the data of the response is not consumed and no 'end' event is emitted.
```